src/routes/(protected)/details/media/[id]/[mediaType]/+page.svelte

```
@@ -53,16 +55,67 @@
+    let showVideoPlayer = $state(page.url.searchParams.get("play") === "true");
     function toggleVideoPlayer() {
         showVideoPlayer = !showVideoPlayer;
     }
 
+    let playParamCleaned = false;
+    $effect(() => {
+        if (!playParamCleaned && showVideoPlayer && page.url.searchParams.has("play")) {
+            playParamCleaned = true;
+            const url = new URL(page.url);
+            url.searchParams.delete("play");
+            replaceState(url, $state.snapshot(page.state));
```

ESLint

[error] Unexpected replaceState() call without resolve().

(svelte/no-navigation-without-resolve)